docker-vertica/packages/cleanup.sh

```diff
@@ -31,7 +32,9 @@ rm -r -f \
    /opt/vertica/oss/python*/lib/python*/site-packages/pip \
    /opt/vertica/oss/python*/lib/python*/config-[0-9]* \
    /opt/vertica/oss/python*/lib/python*/tkinter \
-   /opt/vertica/oss/python*/lib/python*/idlelib
+   /opt/vertica/oss/python*/lib/python*/idlelib \
+   /opt/vertica/oss/python*/lib/python*/site-packages/Cryptodome/SelfTest/PublicKey/test_vectors/ECC \
```

Comment:
As a general fix, could we remove all test directories for packages under site-package? We can match `test` and `*Test`.